buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkIcons.kt

```diff
@@ -147,5 +176,30 @@ private fun toCompose(
           .build()
       )
       .build()
-
   )
+
+private fun toXml(source: BpkIcons, rootDir: String, metadataPath: String): Map<String, String> {
+  fun BpkIcon.fileName() =
+    CaseFormat.LOWER_HYPHEN.to(CaseFormat.LOWER_UNDERSCORE, name)
+      .removeSuffix(".svg")
+      .let {
+        val suffix = if (type == BpkIcon.Type.Sm) "_sm" else ""
+        "bpk_$it$suffix"
+      }
+
+  fun BpkIcon.fileContent(metadata: Map<String, String?>) =
+    value
+      .replace("android:fillColor=\"#FF000000\"", "android:fillColor=\"@color/bpkTextPrimary\"")
+      .let {
+        if (metadata.containsKey(name) && metadata[name] == "true") {
+          it.replaceFirst("\n", "\n    android:autoMirrored=\"true\"\n")
```

Comment:
we need to add the auto mirror property to the vector. This was the cleanest way of doing it without adding an xml parser